deployment/terraform/visualizer/templates/visualizer_conf.tpl

```diff
@@ -0,0 +1,12 @@
+visualizer {
+    metric-consumer {
+      bootstrap.servers = "localhost:19092"
```

Comment:
Bootstrap servers should use `${kafka_endpoint}` here, not a hardcoded host/port.